book/4-clustering.tex

```diff
@@ -56,11 +56,59 @@ \subsection{Pair Confusion Matrix}
 \section{Completeness Score}
 \subsection{Completeness Score}
 
-% ---------- Davis Bouldin Score ----------
+% ---------- Davies Bouldin Score ----------
 \clearpage
 \thispagestyle{clusteringstyle}
-\section{Davis Bouldin Score}
-\subsection{Davis Bouldin Score}
+\section{Davies Bouldin Score}
+\subsection{Davies Bouldin Score}
+
+The Davies-Bouldin score measures the quality of clustering by evaluating the average similarity between each cluster \( C_i \) and its most similar neighboring cluster 
+\( C_j \) for $i,j = 1, 2, ... k$. This similarity \( R_{ij} \) is calculated as the ratio of the within-cluster distance (how tightly packed the cluster is) to the between-cluster distance (how far apart the clusters are).
+
+% The Davies Bouldin score is the average similairty measure of each cluster $C_{i}$ for $i = 1,2,...k$  with its most similar cluster $C_{j}$, where similarity $R_{ij}$ is the ratio of within-cluster distances to between cluster distance. So the clusters which are farther apart and less dispersed will result in a better score. The Davis Bouldin index is defined as 
+
+\begin{center}
+    \tikz{
+    \node[inner sep=2pt, font=\Large] (a) {
+    {
+    $\displaystyle
+    DB = \frac{1}{k} \sum_{{\color{nmlpurple}i}=1}^{k} \max_{{\color{nmlpurple}i} \neq {\color{cyan}j}} R_{{\color{nmlpurple}i}{\color{cyan}j}}
+    $
+    }
+    }; 
+    }
+% \end{center}
+
+% \begin{center}
+    \tikz{
+    \node[inner sep=2pt, font=\Large] (a) {
+    {
+    $\displaystyle
+    R_{{\color{nmlpurple}i}{\color{cyan}j}} = \frac{s_{\color{nmlpurple}i} + s_{\color{cyan}j}}{d_{\color{nmlpurple}i\color{cyan}j}}
+    $
+    }
+    }; 
+    \draw[-latex,nmlpurple, semithick] ($(a.north)+(1.2,0.05)$) to[bend left=15] node[pos=1, right] {cluster diameter} +(1,.5); 
+    \draw[-latex,cyan, semithick] ($(a.south)+(0.6,-0.05)$) to[bend left=15] node[pos=1, left] {distance between centroids} +(-1,-.5); 
+    }
+\end{center}
+
+% A lower Davies-Bouldin score indicates better clustering, as it suggests that clusters are more compact and well-separated from one another.
+% where $s_{i}$ is the average distance between each point of cluster $i$ and centroid of that cluster and $d_{ij}$ is the distance between cluster centroids $i$ and $j$. The minium Davies Bouldin score is zero, with lower values indicating better clustering.
+
+Here, \( s_i \) represents the average distance between each point in cluster \( i \) and the centroid of that cluster, while \( d_{ij} \) is the distance between the centroids of clusters \( i \) and \( j \). The Davies-Bouldin score has a minimum value of zero, with lower scores indicating better-defined clusters that are compact and well-separated.
+
+\textbf{When to use Davies Bouldin?}
+
+Use Davies Bouldin Score when the ground truth labels are unknown and the evaluation must be performed using the model itself.
```

Comment:
I feel this is too general, as this can be said for all clustering evaluation metrics. Maybe mention something like:

Use Davies-Bouldin Score when clusters are roughly spherical and centroid-based. Or in that line.

The idea is to answer the when to use question, as when to use it instead of other metrics.